system-addon/content-src/components/Topics/_Topics.scss

```diff
@@ -39,21 +36,32 @@
   }
 
   .topic-read-more {
+    color: $link-primary;
+
     @media (min-width: $break-point-large) {
-      margin-right: 40px;
       float: right;
+
+      &:dir(rtl) {
+        float: left;
+      }
     }
-    color: $link-primary;
-  }
 
-  .topic-read-more-logo {
-    padding-right: 10px;
-    margin-left: 5px;
-    background-image: url('#{$image-path}topic-show-more-12.svg');
-    background-repeat: no-repeat;
-    fill: $link-primary;
-    -moz-context-properties: fill;
-    vertical-align: middle;
-  }
+    &::after {
+      background-image: url('#{$image-path}topic-show-more-12.svg');
+      background-repeat: no-repeat;
+      content: '';
+      -moz-context-properties: fill;
+      display: inline-block;
+      fill: $link-primary;
+      height: 16px;
+      margin-inline-start: 5px;
+      opacity: 0.8;
```

Comment:
This makes the > look slightly brighter than the topics on my screen. I'd suggest 0.9 or remove, but fine either way :).

Reply:
I'll just remove it. Part of the photon guidelines we were matching had 0.8 opacity for icons, but I think we can just treat this as if it was text.